### ISSUE TITLE
Gemini unsupported

### ISSUE THREAD

**Issue by Andonome** [newsboat/newsboat#1968]
I'd love to see Gemini supported.

I've published an atom feed at gemini://ttrpgs.org/atom.xml, but newsboat doesn't support Gemini.

Perhaps `gemget` could be an optional dependency.

With a simple `gemget ttrpgs.org/atom.xml` the atom feed could be pulled in.

**Comment by dennisschagt** [comment]
I don't expect Gemini support to be added anytime soon, based on https://github.com/newsboat/newsboat/issues/1251#issuecomment-728156718.

However, it is possible to retrieve a feed over Gemini using an `exec:` line in the urls file, e.g.:
```
"exec:gemget ttrpgs.org/atom.xml --output -" "~custom feed title"
```

**Comment by Minoru** [comment]
Yeah, that comment still stands. Please use `exec:` as described above, @Andonome.
